plugins/virus-total.py

```diff
@@ -0,0 +1,111 @@
+""" Virus-Total module to check virus in files """
+
+# By @Krishna_Singhal
+
+import os
+import asyncio
+import requests
+
+from userge import userge, Message, Config
+from userge.utils import progress, humanbytes
+
+API_KEY = os.environ.get("VT_API_KEY", None)
```

Comment:
add this VAR to https://github.com/UsergeTeam/Userge-Plugins/blob/master/config.env.sample